system/doop-central/alerts/infra-frontend/violation.alerts

```diff
@@ -3,6 +3,21 @@
 groups:
   - name: doop-violation.alerts
     rules:
+      # general alert for Critical and Rotten images
+      - alert: GkHighlyVulnerableImages
+        expr: 'sum by (support_group, service) (max_over_time(doop_raw_violations{template_kind="GkVulnerableImages",cluster!~".*qa.*",status=~"Critical|Rotten"}[10m])) > 0'
+        for: 5m
+        labels:
+          context: GkHighlyVulnerableImages
+          support_group: "{{ $labels.support_group }}"
+          service: "{{ $labels.service }}"
+          severity: warning
+          meta: "Highly vulnerable images in prod deployments of {{ title $labels.service }}"
+        annotations:
+          summary: "{{ $count }} pods with highly vulnerable images in prod deployments of {{ title $labels.service }}"
```

Comment:
> system/doop-central/alerts/infra-frontend/violation.alerts: group "doop-violation.alerts", rule 1, "GkHighlyVulnerableImages": annotation "summary": template: __alert_GkHighlyVulnerableImages:1: undefined variable "$count"

Reply:
Do we want to get the value of the matching metric? This would be $value 

https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/